crates/sui-types/src/gas_model/tables.rs

```diff
@@ -49,7 +50,7 @@ pub static INITIAL_COST_SCHEDULE: Lazy<CostTable> = Lazy::new(initial_cost_sched
 ///
 /// Every client must use an instance of this type to interact with the Move VM.
 #[allow(dead_code)]
-#[derive(Debug)]
+#[derive(Debug, Serialize, Deserialize)]
```

Comment:
here and elsewhere for the gas ser/de changes I wonder whether it would be better to extract the info we care from the gas structs rather than deserializing them all. There is a lot of data that we really do not care there.
Not sure...